docs/bigquery-usage.rst

```diff
@@ -1,9 +1,5 @@
-.. toctree::
-  :maxdepth: 0
-  :hidden:
```

Comment:
`.. toctree::` embeds a "local" ToC in a document:  it works especially for long documents w/ many subsections, and even better for documents which `.. include::` others.